### ISSUE TITLE
I encounter a mistake when I use ssh to push file to gitprep

### ISSUE THREAD

**Issue by darkz1984** [yuki-kimoto/gitprep#109]
I encounter a mistake when I use ssh to push file to gitprep
gitprep version 2.3.1

**Comment by yuki-kimoto** [comment]
What do you mean?

**Comment by darkz1984** [comment]
centos 6.5, use key authencaition , after gitprep upgraded, push error and return  code 22

**Comment by darkz1984** [comment]
I use http pull and push ok
when I use public key to pull told me 
"No supported authencation methods available(server send: publickey)"
I had add my key in gitprep.

**Comment by yuki-kimoto** [comment]
I will check public key authentication in my environment at first.

And please see public key, for example, by access the following URL

    /kimoto.keys

**Comment by darkz1984** [comment]
yes,I know how to set the gitprep poublic key setting section,and have set them, I try new gitprep in the test server is OK,but the old upgrade version have the problems...

**Comment by yuki-kimoto** [comment]
does problem occur in only upgraded gitprep?

**Comment by yuki-kimoto** [comment]
@darkz1984

Try delete and add again one public key.
